README.md

```diff
@@ -14,6 +14,110 @@ Create a conda environment with the necessary packages
 
     conda env create -f environment.yml
 
+## Redis example
```

Comment:
These instructions are great!

We many want to consider moving them to the wiki in order to keep the README shorter.